test/SlowTests/Issues/RavenDB-19109.cs

```diff
@@ -33,20 +35,27 @@ public void DatabaseThrowsOnOpenWhenDisableMarkerIsInDirectory()
         var disableMarkerPath = Path.Combine(databasePath, "disable.marker");
         File.Create(disableMarkerPath).Dispose();
 
-        DoCommand(store, false, out exception, false);
-        Assert.NotNull(exception);
-        Assert.Contains(
-            $"Unable to open database: '{store.Database}', it has been manually disabled via the file: '{disableMarkerPath}'. To re-enable, remove the disable.marker and reload the database.",
-            exception.Message);
+        try
+        {
+            DoCommand(store, false, out exception, false);
+            Assert.NotNull(exception);
+            Assert.Contains(
+                $"Unable to open database: '{store.Database}', it has been manually disabled via the file: '{disableMarkerPath}'. To re-enable, remove the disable.marker and reload the database.",
+                exception.Message);
 
 
-        File.Delete(disableMarkerPath);
-        DoCommand(store, false, out exception);
-        Assert.Null(exception);
+            File.Delete(disableMarkerPath);
+            DoCommand(store, false, out exception);
+            Assert.Null(exception);
 
+            {
+                using var session = store.OpenSession();
+                Assert.Equal(1, session.Query<Item>().Count());
+            }
+        }
+        finally
         {
-            using var session = store.OpenSession();
-            Assert.Equal(1, session.Query<Item>().Count());
+            IOExtensions.DeleteFile(disableMarkerPath);
```

Comment:
Shouldn't that have been removed already by the delete directory on db close on tests?
We need to check if we have those on existing directory that will continue to kill the tests.

Reply:
I want to be sure that this gets deleted even if sth fails in the test, hence the tryf. Also using local server.